psalm-baseline.xml

```diff
@@ -2857,6 +2849,19 @@
       <code>objectParameter</code>
     </MissingReturnType>
   </file>
+  <file src="test/TestAsset/Php80Types.php">
+    <ReservedWord occurrences="9">
```

Comment:
Not a problem: these reserved words are valid once we start using PHP 8, and will likely be improved in upstream